packages/framework/presence/src/systemWorkspace.ts

```diff
@@ -30,20 +30,39 @@ export interface SystemWorkspaceDatastore {
 	};
 }
 
-/**
- * There is no implementation class for this interface.
- * It is a simple structure. Most complicated aspect is that
- * `connectionId()` member is replaced with a new
- * function when a more recent connection is added.
- *
- * See {@link SystemWorkspaceImpl.ensureAttendee}.
- */
-interface SessionClient extends ISessionClient {
-	/**
-	 * Order is used to track the most recent client connection
-	 * during a session.
-	 */
-	order: number;
+class SessionClient implements ISessionClient {
+	public readonly sessionId: ClientSessionId;
+	public order: number;
+	private _connectionId: ClientConnectionId | undefined;
+	private _connectionStatus: SessionClientStatus;
```

Comment:
No underscores needed. For connectionId, you can declare directly in constructor. No assignment or separate declaration needed.